cli/app.go

```diff
@@ -2479,6 +2479,24 @@ Note: There is no progress meter while copying is in progress.
 			UsageText:       createUsageText("module", nil, false, true),
 			HideHelpCommand: true,
 			Subcommands: []*cli.Command{
+				{
+					Name:      "local-app-testing",
+					Usage:     "test your viam application locally",
+					UsageText: createUsageText("module local-app-testing", []string{"port", "app-url"}, false, false),
+					Flags: []cli.Flag{
+						&cli.IntFlag{
+							Name:  "port",
+							Usage: "port to run the local server on (default: 8000)",
+							Value: 8000,
+						},
+						&cli.StringFlag{
+							Name:     "app-url",
+							Usage:    "url where local app is running",
+							Required: true,
+						},
```

Comment:
[q] why do we need this?

Reply:
feedback from sync emily call: can we make them both port otherwise it seems like theyre more connected